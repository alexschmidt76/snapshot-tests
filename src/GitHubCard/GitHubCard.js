import React from "react";
import { Card } from "react-bootstrap";
import avatar from './biggie_cheese.jpg';

export default function GitHubCard() {
    return (
        <Card>
            <Card.Img variant='top' src={avatar} />
            <Card.Title>Alex Schmidt</Card.Title>
            <Card.Text>
                My name is Alex and I love music and video games. <br />
                I am currently studying React in a coding bootcamp! 
            </Card.Text>
        </Card>
    );
}