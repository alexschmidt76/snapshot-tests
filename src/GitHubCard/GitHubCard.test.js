import renderer from 'react-test-renderer';
import React from 'react';
import GitHubCard from './GitHubCard';

test('renders a snapshot', () => {
    const tree = renderer.create(<GitHubCard />).toJSON()
    expect(tree).toMatchSnapshot();
});