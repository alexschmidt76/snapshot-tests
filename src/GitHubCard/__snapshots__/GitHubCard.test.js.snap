// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a snapshot 1`] = `
<div
  className="card"
>
  <img
    className="card-img-top"
    src="biggie_cheese.jpg"
  />
  <div
    className="card-title h5"
  >
    Alex Schmidt
  </div>
  <p
    className="card-text"
  >
    My name is Alex and I love music and video games. 
    <br />
    I am currently studying React in a coding bootcamp!
  </p>
</div>
`;
